### PR TITLE
update celery and kombu

### PR DESCRIPTION
**Pull request by auvipy**
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Review by turbaszek — APPROVED** [review]

**Comment by auvipy** [comment]
this release fixes some regression.

**Comment by turbaszek** [comment]
@kaxil @potiuk should we merge or run all tests?

**Comment by potiuk** [comment]
Yeah.  Adding an artificial plugin change for now should do the trick. Alternatively (And possibly better)  add requirements/* to the regexp here; https://github.com/apache/airflow/blob/master/.github/workflows/ci.yml#L169

**Comment by turbaszek** [comment]
@auvipy once this one https://github.com/apache/airflow/pull/9690 will you be able to rebase?

**Comment by auvipy** [comment]
sure just ping me

**Comment by turbaszek** [comment]
@auvipy would you mind rebasing?

**Comment by auvipy** [comment]
do we need that? it's showing all are passing

**Comment by turbaszek** [comment]
> do we need that? it's showing all are passing

Only static checks has passed. To optimize the builds we do not run tests with "docs" changes. That was fixed in #9690

**Review by turbaszek — APPROVED** [review]

**Review by auvipy — COMMENTED** [review]
in the mean time, I released another bug fix release of celery==4.4.7

File: requirements/requirements-python3.8.txt [hunk 1]
```diff
@@ -80,7 +80,7 @@ cached-property==1.5.1
 cachetools==4.1.0
 cassandra-driver==3.20.2
 cattrs==1.0.0
-celery==4.4.5
+celery==4.4.6
```
Comment (turbaszek) [inline requirements/requirements-python3.8.txt]:
```suggestion
celery==4.4.7
```
@auvipy should we try?

**Pull request merged**